webapp/store/login/validator.js

```diff
@@ -12,42 +12,47 @@ const getProp = (propName, defaultValue) => R.pathOr(defaultValue, propName.spli
 
 const _validatePassword = (propName, item) => {
   const password = getProp(propName)(item)
-  return !validPasswordRe.test(password)
-    ? { key: Validation.messageKeys.user.passwordInvalid }
-    : !passwordStrengthRe.test(password)
-    ? { key: Validation.messageKeys.user.passwordUnsafe }
-    : null
+
+  if (!validPasswordRe.test(password)) {
+    return { key: Validation.messageKeys.user.passwordInvalid }
+  }
+
+  if (!passwordStrengthRe.test(password)) {
+    return { key: Validation.messageKeys.user.passwordUnsafe }
+  }
```

Comment:
We can group the regular expressions and avoid using abbreviation
```
const regex = {
  validPassword: new RegExp(/^[\S]+.*[\S]+$/),
  // It can also be shorten like so
  passwordStrength: /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?\d).{8,}$/,
}
// usage:
regex.passwordStrength.test(string)
```

Reply:
can we address this later? I will add a separate issue